targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/cycfg_qspi_memslot.cyqspi

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```

Comment:
why is this file need in the tree , is it used?

Reply:
@0xc0170  This file is used by our tools to configure the QSPI connections on our configurable devices. This was added in anticipation of our tools support the Mbed OS flow this quarter, allowing a customer to customize how the QSPI interface of the P64 device is configured.